test/unit/org/apache/cassandra/io/sstable/metadata/StatsMetadataTest.java

```diff
@@ -0,0 +1,36 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.apache.cassandra.io.sstable.metadata;
+
+import org.junit.Test;
+
+import org.apache.cassandra.utils.EstimatedHistogram;
+
+public class StatsMetadataTest
+{
+    @Test
+    public void shouldHandleUnusablePartitionCellHistogram()
+    {
+        EstimatedHistogram histogram = new EstimatedHistogram(3);
+        histogram.add(10);
+        StatsMetadata metadata = new StatsMetadata(null, histogram, null, -1L, -1L, -1, -1, -1, -1, 0.0f, null, -1, 
+                                                   null, null, false, -1L, -1L, -1L, null, false);
+        metadata.getEstimatedDroppableTombstoneRatio(0);
```

Comment:
Nit: I would assert that the `histogram.isOverflowed` is `true` and check that the returned value of `getEstimatedDroppableTombstoneRatio` is the expected one to make the test a bit more robuste.